src/servers/src/mysql/handler.rs

```diff
@@ -112,10 +114,12 @@ impl MysqlInstanceShim {
                 Default::default(),
                 // TODO(sunng87): generate process id properly
                 0,
+                Some(connection_id),
```

Comment:
Can we just use this connection_id for process_id, the `0` above?

Reply:
The mysql connection id is exact the process id: https://dev.mysql.com/doc/refman/8.4/en/information-functions.html#function_connection-id

Reply:
Yes. I suggest to use u32 for process id. In postgres it's 32 bit integer too.

Reply:
> Yes. I suggest to use u32 for process id. In postgres it's 32 bit integer too.

fixed in https://github.com/GreptimeTeam/greptimedb/pull/6320/commits/8f300eeb3f343e0b4a0035ae0e0ce6d507a9d362